<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkLabelSetWidgetControls</class>
 <widget class="QWidget" name="QmitkLabelSetWidgetControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>397</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <widget class="QToolButton" name="m_btNewLabelSet">
        <property name="toolTip">
         <string>Create a new segmentation session</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../../resources/multilabelsegmentation.qrc">
          <normaloff>:/multilabelsegmentation/NewLabelSet_48.png</normaloff>:/multilabelsegmentation/NewLabelSet_48.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_btLoadLabelSet">
        <property name="toolTip">
         <string>Load a segmentation session</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../../resources/multilabelsegmentation.qrc">
          <normaloff>:/multilabelsegmentation/LoadLabelSet_48.png</normaloff>:/multilabelsegmentation/LoadLabelSet_48.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_btSaveLabelSet">
        <property name="toolTip">
         <string>Save current segmentation session</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../../resources/multilabelsegmentation.qrc">
          <normaloff>:/multilabelsegmentation/SaveLabelSet_48.png</normaloff>:/multilabelsegmentation/SaveLabelSet_48.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_btImportLabelSet">
        <property name="toolTip">
         <string>Import labels from another segmentation session</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../../resources/multilabelsegmentation.qrc">
          <normaloff>:/multilabelsegmentation/ImportLabelSet_48.png</normaloff>:/multilabelsegmentation/ImportLabelSet_48.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="m_btNewLabel">
        <property name="toolTip">
         <string>Add a new label to currently active segmentation</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../../resources/multilabelsegmentation.qrc">
          <normaloff>:/multilabelsegmentation/NewLabel_48.png</normaloff>:/multilabelsegmentation/NewLabel_48.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QmitkLabelSetTableWidget" name="m_LabelSetTableWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ctkSearchBox" name="m_LabelSearchBox"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkLabelSetTableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">QmitkLabelSetTableWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ctkSearchBox</class>
   <extends>QLineEdit</extends>
   <header location="global">ctkSearchBox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../resources/multilabelsegmentation.qrc"/>
 </resources>
 <connections/>
</ui>
